tracer/src/Datadog.Trace/AppSec/Rcm/ConfigurationStatus.cs

```diff
@@ -0,0 +1,147 @@
+// <copyright file="ConfigurationStatus.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.AppSec.Rcm.Models.Asm;
+using Datadog.Trace.AppSec.Rcm.Models.AsmData;
+using Datadog.Trace.AppSec.Rcm.Models.AsmDd;
+using Datadog.Trace.AppSec.Waf.Initialization;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using Action = Datadog.Trace.AppSec.Rcm.Models.Asm.Action;
+
+namespace Datadog.Trace.AppSec.Rcm;
+
+internal record ConfigurationStatus
```

Comment:
This is the class that I would change to a ConfigurationUpdater (but again not in this PR)